public/app/plugins/datasource/prometheus/querybuilder/components/promQail/state/helpers.ts

```diff
@@ -163,13 +151,18 @@ export async function promQailSuggest(
   // when you're not running promqail
   const check = (await llms.openai.enabled()) && (await llms.vector.enabled());
 
-  if (!check) {
+  const interactionToUpdate = interaction ? interaction : createInteraction(SuggestionType.Historical);
+
+  if (!check || interactionToUpdate.suggestionType === SuggestionType.Historical) {
     return new Promise<void>((resolve) => {
       return setTimeout(() => {
-        const interactionToUpdate = interaction ? interaction : createInteraction(SuggestionType.Historical);
 
-        const suggestions =
-          interactionToUpdate.suggestionType === SuggestionType.Historical ? querySuggestions : [querySuggestions[0]];
+        const suggestions = commonTemplateSuggestions.map(suggestion => {
+          return {
+            query: suggestion.replace('metric', query.metric),
```

Comment:
this throws away the labels the user may have chosen. Deliberate just to simplify the result?